cluster/log-dump/README.md

```diff
@@ -0,0 +1,14 @@
+## This directory is deprecated!
+
+We have recently ported the log dumping utility from kubernetes/kubernetes repository to
+[kubernetes/test-infra](https://github.com/kubernetes/test-infra/tree/master/logexporter/cluster).
+If you require changes to this script, please consider migrating your jobs to use the new
+log dumping mechanism first.
+
+## Migration steps
+
+Currently, `log-dump.sh` file is added to every newly released `kubekins-e2e` image.
+In order to leverage that script, add `USE_KUBEKINS_LOG_DUMPING` environment variable
+to your test job and set its value to `true`.
+
+For the time being, only GCE and GKE providers are supported by the new log dumping mechanism.
```

Comment:
I would rephrase this to encourage adoption, saying that only GCE and GKE were migrated so far and specifying migrations steps for the rest.

Reply:
Could you point to more specific place in kubetest?

Also it seems that old log dumping also supported only gce/gke. Could we drop "old" from the first sentence then? Maybe replacing with `are supported by the log-dump mechanism.`?

Reply:
Done.